Python/pdf_reorder/README.md

```diff
@@ -0,0 +1,36 @@
+
+## A PDF reordering script
+
+This scrip can be used for reordering a pdf file which may have pages in wrong order.
+
+We used **pdfrw** library for this script. 
+
+A more efficient,faster alternative is [PyMuPDF](https://pypi.org/project/PyMuPDF/).       
+The PyMuPDF library can have dependency issues with versions of visual C++ hence we used a more simpler library pdfrw.
+
+ Download pdfrw using
+	```pip install pdfrw```
```

Comment:
Remove this, and add a `requirements.txt` to download all packages automatically.